src/main/java/org/lareferencia/core/harvester/workers/HarvestingWorker.java

```diff
@@ -137,7 +140,34 @@ public void stop() {
 	 */
 	@Override
 	public void run() {
-		
+
+		// create identify map for contain the information of identify request
+		Map<String, String> identifyMap = null;
+
+		// date granularity
+		String granularity = DEFAULT_GRANDULARITY;
+
+		// check if the url is valid by trying to connect to it with identify verb
+		String originURL = runningContext.getNetwork().getOriginURL();
+
+
+		// if fetch identify is true, will try to fetch the identify information
+		if ( runningContext.getNetwork().getBooleanPropertyValue("HARVEST_IDENTIFY_PARAMETERS") ) {
```

Comment:
Does this makes sense? You are given the admin user the responsability to understand how the OAI of the repository works and add this setting on a different screen, you are creating an usability issue, the person when defining the OAI, must also remember to go to a different screen to check the required configuration. I don't think it's very praticle, when, like us, you have more than 400 resources.